Modules/Relationships/src/Relationships.Domain/DomainEvents/Outgoing/RelationshipDecomposedDomainEvent.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Backbone.BuildingBlocks.Domain.Events;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Relationships.Domain.Aggregates.Relationships;
+
+namespace Backbone.Modules.Relationships.Domain.DomainEvents.Outgoing;
+
+public class RelationshipDecomposedDomainEvent : DomainEvent
```

Comment:
I can't find a usage of this. Am I missing something?

Reply:
I think I missed something. Changes made.